### PR TITLE
Added support for multiple parts with same name in oatpp::web::mime::multipart::PartList

### PR DESCRIPTION
**Pull request by MHaselmaier**
Implements  #534

### CODE REVIEW

File: src/oatpp/web/mime/multipart/PartList.hpp [hunk 1]
```diff
@@ -74,12 +75,21 @@ class PartList : public Multipart {
...
    * @param name - &id:oatpp::String;.
    * @return - &id:oatpp::web::mime::multipart::Part;.
    */
   std::shared_ptr<Part> getNamedPart(const oatpp::String& name);
 
+  /**
+   * Get all parts by name <br>
+   * Applicable to named parts only.
+   * @param name - &id:oatpp::String;.
+   * @return - std::list<std::shared_ptr<&id:oatpp::web::mime::multipart::Part;>>.
```
Comment (lganzzzo) [inline src/oatpp/web/mime/multipart/PartList.hpp:89]: please escape triangle brackets:

```
@return - std::list<std::shared_ptr<&id:oatpp::web::mime::multipart::Part;>>.
```

change to

```
@return - std::list&lt;std::shared_ptr&lt;&id:oatpp::web::mime::multipart::Part;&gt;&gt;.
```

**Review by lganzzzo — COMMENTED** [review]
Looks good.
Please fix the API docs for `getNamedParts` method and it's ready to be merged

File: src/oatpp/web/mime/multipart/PartList.hpp (hunk 1 above)
Comment (MHaselmaier) [reply src/oatpp/web/mime/multipart/PartList.hpp]: I used the style of `getAllParts`

**Review by lganzzzo — APPROVED** [review]

**Comment by lganzzzo** [comment]
Merged!

**Pull request merged**